tasks/collect-rh-sign-image-params/collect-rh-sign-image-params.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-rh-sign-image-params
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/tags: release
+spec:
+  params:
+    - description: An image of operator-pipeline-images for the steps to run in.
+      name: pipeline_image
+      default: "quay.io/redhat-isv/operator-pipelines-images:released"
```

Comment:
This what we've been using in internal-services since the beginning. If we want to hamonize between both this and internal services, let's open a ticket to find out what the right image and fix it everywhere.

Reply:
> This what we've been using in internal-services since the beginning.

Yeah, thinking out loud: I was gonna say the potential risk here is that now you're copying the task definition here. Until now, we consumed their tasks directly via tekton bundles. But in that setup, we pointed to a fixed task bundle which pointed to this `:released` image. And they used the tasks as well. If they did a breaking change in the image, they would likely update the tasks. But would they tell us to move to the new task bundle version to avoid breakage? Not sure... So either way, you could be broken. We have this Jira: https://issues.redhat.com/browse/RELEASE-1207
And I was supposed to find out if we should also make our copies of the tasks. I am not really sure...